src/librustc/middle/lang_items.rs

```diff
@@ -338,6 +338,8 @@ language_item_table! {
     U128ShloFnLangItem,              "u128_shlo",               u128_shlo_fn;
     I128ShroFnLangItem,              "i128_shro",               i128_shro_fn;
     U128ShroFnLangItem,              "u128_shro",               u128_shro_fn;
+
```

Comment:
is this line expected?

Reply:
Yeah, I thought because there are multiple empty lines (scroll a little bit up in the file).